// [[Reason: Implicit undefined not fixed by code mod yet]] ////

//// [tests/cases/compiler/declarationEmitParameterProperty.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,8 +1,8 @@
 
 
 //// [declarationEmitParameterProperty.d.ts]
 export declare class Foo {
-    bar?: string | undefined;
-    constructor(bar?: string | undefined);
+    bar?: string;
+    constructor(bar?: string);
 }
 //# sourceMappingURL=declarationEmitParameterProperty.d.ts.map
\ No newline at end of file
